### PR TITLE
[Renaming HB] Renaming Harbourbridge to Spanner migration tool - Command

### PR DESCRIPTION
**Pull request by darshan-sj**
[Renaming HB] Renaming Harbourbridge to Spanner migration tool - Command and docker file changes

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -12,40 +12,41 @@
...
+# This directory binds with the host directory that is to store the output files generated by Spanner migration tool.
+WORKDIR /spanner_migration_tool/spanner_migration_tool_output
+# Create directory to store the Spanner migration tool binary. 
+RUN mkdir /spanner_migration_tool/bin
 # Create directory to store the dump files. This directory binds with the host directory that contains the dump files of the source database.
-RUN mkdir /harbourbridge/source_dump
-# Copy the HarbourBridge binary from the build stage's harbourbridge_bin directory to the release stage's harbourbridge/bin directory.
-COPY --from=build /harbourbridge_bin/harbourbridge /harbourbridge/bin
+RUN mkdir /spanner_migration_tool/source_dump
+# Copy the spanner-migration-tool binary from the build stage's spanner_migraiton_tool_bin directory to the release stage's spanner_migration_tool/bin directory.
```
Comment (shreyakhajanchi) [inline Dockerfile:48]: nit spell check: spanner_migraiton_tool_bin  to spanner_migration_tool_bin

**Review by shreyakhajanchi — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (darshan-sj) [reply Dockerfile]: Done.

**Pull request merged**